api/WindowOrWorkerGlobalScope.json

```diff
@@ -858,7 +858,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": "2.0"
+              "version_added": "1.5"
```

Comment:
This seems like a random change, but indeed Chrome 24 would mirror to Samsung 1.5.